src/cdt.rs

```diff
@@ -345,7 +345,42 @@ where
     ///
     /// Panics if any constraint edges overlap. Panics if the edges contain an invalid index (out of range).
     pub fn bulk_load_cdt(vertices: Vec<V>, edges: Vec<[usize; 2]>) -> Result<Self, InsertionError> {
-        let mut result = bulk_load_cdt(vertices, edges)?;
+        let mut result = bulk_load_cdt(vertices, edges).unwrap();
+        *result.hint_generator_mut() = L::initialize_from_triangulation(&result);
+        Ok(result)
+    }
+
+    /// Same behaviour as [bulk_load_cdt], but rather than panicking,
+    /// ignores and calls the parameter function `on_conflict_found` for each conflicting edges encountered.
+    ///
+    /// # Example
+    /// ```
+    /// # fn main() -> Result<(), spade::InsertionError> {
+    /// use spade::{ConstrainedDelaunayTriangulation, Point2, Triangulation};
+    /// let mut vertices = vec![
+    ///     Point2::new(0.0, 1.0),
+    ///     Point2::new(1.0, 2.0),
+    ///     Point2::new(3.0, -3.0),
+    ///     Point2::new(-1.0, -2.0),
+    ///     Point2::new(-4.0, -5.0),
+    /// ];
+    /// let mut conflicting_edges = Vec::new();
+    /// let mut edges = vec![[0, 1], [1, 2], [2, 3], [3, 4]];
+    /// let cdt = ConstrainedDelaunayTriangulation::<_>::try_bulk_load_cdt(vertices.clone(), edges, |e| conflicting_edges.push(e))?;
+    ///
+    /// assert_eq!(cdt.num_vertices(), 5);
+    /// assert_eq!(cdt.num_constraints(), 4);
+    /// // The order will usually change
+    /// assert_ne!(cdt.vertices().map(|v| v.position()).collect::<Vec<_>>(), vertices);
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn try_bulk_load_cdt(
```

Comment:
I took the initiative to keep api consistent everywhere with a function call, and user is responsible of storing the conflicting edges if desirable ; I think it makes more sense than providing higher level API, with an example code doing that.

I'm not sure if this particular API is enough for user to be able to modify and insert an edge when encountering a conflict, it may be a good idea to pass some additional data to enable this, or provide yet another endpoint `try_bulk_load_cdt_or_modify` ?